downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CreateGOAFile.java

```diff
@@ -0,0 +1,58 @@
+package org.reactome.release.downloadDirectory.GenerateGOAnnotationFile;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.*;
+
+public class CreateGOAFile {
+
+    private static final Logger logger = LogManager.getLogger();
+
+    private static final String CELLULAR_COMPONENT = "Cellular Component";
+    private static final String MOLECULAR_FUNCTION = "Molecular Function";
+    private static final String BIOLOGICAL_PROCESS = "Biological Process";
+
+    private static final String GOA_FILENAME = "gene_association.reactome";
+
+    public static void execute(MySQLAdaptor dbAdaptor, String releaseNumber) throws Exception {
+        logger.info("Generating GO annotation file: gene_association.reactome");
+
+        for (GKInstance reactionInst : (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent)) {
+            // Only finding GO accessions from curated ReactionlikeEvents
+            if (!isInferred(reactionInst)) {
+                CellularComponentAnnotationBuilder.processCellularComponents(reactionInst);
+                MolecularFunctionAnnotationBuilder.processMolecularFunctions(reactionInst);
+                BiologicalProcessAnnotationBuilder.processBiologicalFunctions(reactionInst);
+            }
+        }
+        GOAGeneratorUtilities.outputGOAFile();
+        Files.move(Paths.get(GOA_FILENAME), Paths.get(releaseNumber + "/" + GOA_FILENAME), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
To make it more declarative, can this functionality be moved to the Utilities class or be a private method like:
```
private static void moveFile(String filePath, String targetDirectory) {
    Files.move(Paths.get(filePath), Paths.get(targetDirectory), StandardCopyOption.REPLACE_EXISTING);
} 
```